jquery.flot.time.js

```diff
@@ -278,11 +280,21 @@ API.txt for details.
     var specQuarters = baseSpec.concat([[1, "quarter"], [2, "quarter"],
         [1, "year"]]);
 
+    var axisFirstData; //record first value of the datapoints
```

Comment:
make sure that you think about XY data, analog waveforms, negative relative time ...